freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -280,6 +280,7 @@ class ForceBuyPayload(BaseModel):
     price: Optional[float]
     ordertype: Optional[OrderTypeValues]
     stakeamount: Optional[float]
+    buy_tag: Optional[str]
```

Comment:
In light of current proceedings in `feat/short` - i'd ask you to name this `entry_tag` instead.

while it'll create a short-term misalignment (the API is sending `entry_tag` - but it end's up in buy_tag, and the client get's "buy_tag" back) - i think it's better to have it right immediately, rather than renaming it a month later.

Reply:
sure, just updated it.